### DISCUSSION TITLE
求助，怎么修改头像和姓名啊

### DISCUSSION THREAD

**Discussion by rqrq201704** [share121/inter-knot#5216] [委托]
怎么修改头像和姓名啊，我不想顶着一个字符串

**Comment by Czdmk** [comment]
在网页版GitHub里可以修改![IMG_20240920_020923.jpg](https://github.com/user-attachments/assets/419b72e9-f11b-4a01-abca-97fa00d23d49)

**Reply by emmGrinch** [reply]
不过APP的头像分辨率有点低<img src="https://bbs-static.miyoushe.com/static/2024/09/10/ed624befad4ed330fbdb41c91fc09d10_3535401542533232063.png" alt="飞霄 夸夸" width="75" >

**Comment by Kazuha-hhh** [comment]
先随便找个帖子点击进行评论，接着在右上角找到自己头像并点击进入个人空间，接着利用工具翻译英文自己摸索吧

**Comment by flystarnx** [comment]
在网页改可以<img src="https://upload-bbs.mihoyo.com/upload/2022/11/14/4b4fb49d4bd402c530600e758ebb06bd_5477506439621383475.png" alt="阿姬开心" width="75" >

**Comment by LutionDDT** [comment]
试试
